lib/features/sks_people_live/data/repository/latest_sks_user_data_repo.dart

```diff
@@ -1,14 +1,34 @@
+import "dart:async";
 import "package:flutter_riverpod/flutter_riverpod.dart";
 import "package:riverpod_annotation/riverpod_annotation.dart";
 import "../../../../shared_api_clients/sks_api_client.dart";
 import "../models/sks_user_data.dart";
 
 part "latest_sks_user_data_repo.g.dart";
 
+extension RefIntervalRefreshX on Ref {
+  void setRefresh(Duration interval) {
+    final timer = Timer.periodic(
+      interval,
+      (t) => invalidateSelf(),
+    );
+    onDispose(timer.cancel);
+  }
+}
+
```

Comment:
We have this part of code in file parkings_repository.dart so my idea is to move this to separate file(like ref_extesions.dart) in utils folder and use it in 2 places.

Reply:
Sure thing! I can take care of refctorization. By the way - to test code it is nessesary to add SksAppBar somwhere. It wasn't placed anywhere before so I left it as it was.

Reply:
You can add it at app bar at main screen (for testing purposes)

Reply:
good catch Tomuś